tests/providers/gcp/gcp_test.py

```diff
@@ -634,8 +695,75 @@ def testExecuteQuery(self, mock_gcl_api):
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.storage.GoogleCloudStorage.GcsApi')
+  def testGetOperationObject(self, mock_gcs_api):
+    """Test GCS object Get operation."""
+    api_get_object = mock_gcs_api.return_value.objects.return_value.get
+    api_get_object.return_value.execute.return_value = MOCK_GCS_OPERATION_OBJECT_GET
+    get_results = FAKE_GCS.GetOperationObject('gs://Fake_Path')
+    self.assertEqual(get_results, MOCK_GCS_OPERATION_OBJECT_GET)
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.storage.GoogleCloudStorage.GcsApi')
+  def testGetMD5Object(self, mock_gcs_api):
+    """Test GCS object MD5 hash."""
+    api_get_object = mock_gcs_api.return_value.objects.return_value.get
+    api_get_object.return_value.execute.return_value = MOCK_GCS_OPERATION_OBJECT_GET
+    md5_base64 = FAKE_GCS.GetMD5Object('gs://Fake_Path')
+    self.assertEqual(md5_base64, MOCK_GCS_OPERATION_OBJECT_GET['md5Hash'])
+    md5_hex = FAKE_GCS.GetMD5Object('gs://Fake_Path', in_hex=True)
+    self.assertEqual(md5_hex, base64.b64decode(MOCK_GCS_OPERATION_OBJECT_GET['md5Hash']).hex())
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.storage.GoogleCloudStorage.GcsApi')
+  def testGetSizeObject(self, mock_gcs_api):
+    """Test GCS object size."""
+    api_get_object = mock_gcs_api.return_value.objects.return_value.get
+    api_get_object.return_value.execute.return_value = MOCK_GCS_OPERATION_OBJECT_GET
+    object_size = FAKE_GCS.GetSizeObject('gs://Fake_Path')
+    self.assertEqual(object_size, MOCK_GCS_OPERATION_OBJECT_GET['size'])
+
+
+class GoogleCloudBuildeTest(unittest.TestCase):
+  """Test Google Cloud Build class."""
+  # pylint: disable=line-too-long
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.build.GoogleCloudBuild.GcbApi')
+  def testCreateBuild(self, mock_gcb_api):
+    """Test that Cloud Builds are correctly created."""
+    build_create_object = mock_gcb_api.return_value.projects.return_value.builds.return_value.create
+    build_create_object.return_value.execute.return_value = MOCK_GCB_BUILDS_CREATE
+    build_response = FAKE_GCB.CreateBuild({'Fake-Build_body': None})
+    self.assertEqual(build_response, MOCK_GCB_BUILDS_CREATE)
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.build.GoogleCloudBuild.GcbApi')
+  def testBlockOperation(self, mock_gcb_api):
+    """Test that Cloud Builds are correctly blocked until done."""
+    build_operation_object = mock_gcb_api.return_value.operations.return_value.get
+    build_operation_object.return_value.execute.return_value = MOCK_GCB_BUILDS_SUCCESS
+    block_build_success = FAKE_GCB.BlockOperation(MOCK_GCB_BUILDS_CREATE)
+    self.assertEqual(block_build_success, MOCK_GCB_BUILDS_SUCCESS)
+    build_operation_object.return_value.execute.return_value = MOCK_GCB_BUILDS_FAIL
+    with self.assertRaises(RuntimeError):
```

Comment:
To stay consistent with other tests, wdyt about changing this to:

```
self.assertRaises(RuntimeError, FAKE_GCB.BlockOperation, MOCK_GCB_BUILDS_CREATE)
```

I don't know if this makes things better or worst to be honest, we could also change the other assertions to the `with` format in a separate PR. Thoughts @tomchop @hacktobeer ?

Reply:
as discussed let's use assertRaises as a context manager (+ with key word) it makes it more explicit and the others we change in issue/173.